Securitytxt/Model/Config/Signature.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * This file is part of the Kalpesh_Securitytxt module.
+ *
+ * @author      Kalpesh Mehta <k@lpe.sh>
+ * @copyright   Copyright (c) 2018-2019
+ *
+ * For full copyright and license information, please check the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Kalpesh\Securitytxt\Model\Config;
+
+use \Magento\Config\Model\Config\CommentInterface;
+
+class Signature implements CommentInterface
+{
+    /**
+     * @var \Magento\Store\Model\StoreManagerInterface
+     */
+    protected $_storeManagerInterface;
```

Comment:
By the newest convention we do not prefix private properties with underscore. Also, it's highly discouraged to use `protected` visibility in core code. It may be changed to `private` here and in all other places.